src/licensedcode/data/licenses/cfitsio.LICENSE

```diff
@@ -0,0 +1,32 @@
+---
+key: cfitsio
+short_name: CFITSIO License
+name: CFITSIO License
+category: Permissive
+owner: NASA
+spdx_license_key: CFITSIO
+other_urls:
+    - https://heasarc.gsfc.nasa.gov/docs/software/fitsio/c/f_user/node9.html
+---
+
+Copyright (Unpublished-all rights reserved under the copyright laws of the United States),
```

Comment:
We have a perfect detection on this one and this seems to be a combination of `us-govt-public-domain AND nist-pd`
```
{
      "identifier": "us_govt_public_domain_and_nist_pd-42047a8f-ce7a-7bb5-806f-354563734159",
      "license_expression": "us-govt-public-domain AND nist-pd",
      "count": 1,
      "detection_log": [
        "not-combined"
      ],
      "matches": [
        {
          "score": 100.0,
          "start_line": 307,
          "end_line": 307,
          "matched_length": 14,
          "match_coverage": 100.0,
          "matcher": "2-aho",
          "license_expression": "us-govt-public-domain",
          "rule_identifier": "us-govt-public-domain_27.RULE",
          "rule_url": "https://github.com/nexB/scancode-toolkit/tree/develop/src/licensedcode/data/rules/us-govt-public-domain_27.RULE"
        },
        {
          "score": 100.0,
          "start_line": 309,
          "end_line": 313,
          "matched_length": 177,
          "match_coverage": 100.0,
          "matcher": "2-aho",
          "license_expression": "nist-pd",
          "rule_identifier": "nist-pd_14.RULE",
          "rule_url": "https://github.com/nexB/scancode-toolkit/tree/develop/src/licensedcode/data/rules/nist-pd_14.RULE"
        }
      ]
    },
```